benchmark-timed-schedule.yaml

```diff
@@ -1,15 +1,37 @@
 ##############################################################################
 # YAML Anchors
 ##############################################################################
-common_infrastructure_params: &common_infrastructure_params
+infrastructure_vars: &infrastructure_vars
   TF_VAR_k8s_min_master_version: ((k8s_min_master_version))
   TF_VAR_gcp_folder_id: ((gcp_folder_id))
   TF_VAR_gcp_billing_account: ((gcp_billing_account))
   TF_VAR_project_env: ((gcp_project_label))
+  TF_VAR_machine_type: ((machine_type))
   PROJECT_NAME: ((project_name))
   SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
   GOOGLE_APPLICATION_CREDENTIALS: /root/gcloud-service-key.json
 
+deploy_infrastructure_vars: &deploy_infrastructure_vars
+  IMPORT_EXISTING_PROJECT: ((import_existing_project))
+
+destroy_infrastructure_vars: &destroy_infrastructure_vars
+  LOAD_GENERATOR_TF_STATE_BUCKET: ((load_generator_tf_state_bucket))
+
+run_locust_vars: &run_locust_vars
```

Comment:
This should be `run_benchmark` rather than `run_locust` to be consistent with the job name.